data/core/terrain.cfg

```diff
@@ -1276,7 +1276,9 @@ Occasionally caves are <italic>text='illuminated'</italic>."
     #editor_image=cave/earthy-wall-hewn-tile
     id=cavewall_earthy_hewn
     name= _ "Cave Wall"
-    editor_name="Reinforced Earthy Cave Wall" # wmllint: ignore
+#ifndef SCHEMA_SHOULD_SKIP_THIS
+    editor_name="Reinforced Earthy Cave Wall"
```

Comment:
This should be translatable.

Reply:
The reason I left it untranslated was because it's deprecated… although if there's a non-deprecated terrain with the same name, I may as well just make it translated…